### ISSUE TITLE
Add support for Ubuntu 20.04

### ISSUE THREAD

**Issue by jdrch** [Sysinternals/ProcMon-for-Linux#51]
See title. Manually setting up the repo and trying to install on 20.04 results in: 

```
# apt install procmon
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package procmon
```

**Comment by the-ress** [comment]
related: https://github.com/microsoft/ProcMon-for-Linux/issues/25#issuecomment-663891568 and #2

**Comment by trivalik** [comment]
Build from master BCC via `git clone https://github.com/iovisor/bcc.git` and master of ProcMon-for-Linux makes it working for 20.10, probably also for your version

**Comment by villelaitila** [comment]
That really helped with my issue. But not before I did `sudo apt remove libclang-common-10-dev && sudo apt autoremove` to get rid of conflicting LLVM 10 with its own incompatible version of clang. Dockerized setup would actually be very nice for this tool!

**Comment by jahabibi** [comment]
Resolved in [Procmon 1.0.1](https://github.com/Sysinternals/ProcMon-for-Linux/tree/1.0.1)
